data/json/recipes/ammo/components.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "result": "incendiary",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_COMPONENTS",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": 15000,
+    "book_learn": [ [ "textbook_chemistry", 5 ], [ "recipe_bullets", 3 ], [ "textbook_anarch", 0 ] ],
+    "qualities": [ { "id": "CHEM", "level": 2 } ],
+    "tools": [ [ [ "hotplate", 50 ], [ "toolset", 50 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "magnesium", 100 ] ],
```

Comment:
If it uses an uncraftable rare drop now, shouldn't it lose all the other heavy requirements?
All the chem stuff was for turning sugar into incendiary, magnesium is better at it from the start.